.github/workflows/e2e.yml

```diff
@@ -253,7 +253,7 @@ jobs:
           - make-target: "start-upgrade-test"
             runs-on: ubuntu-22.04
             run: ${{ needs.matrix-conditionals.outputs.UPGRADE_TESTS == 'true' }}
-          - make-target: "start-upgrade-test-zetaclient"
+          - make-target: "start-upgrade-test-zetaclient-light"
```

Comment:
We now have different versions of zetaclient and zetacore. We have an option to run zetae2e using either the current version or the older version, but both throw a different set of errors .
In my opinon for the zetaclient only migration we dont really need to test with both test runs as this only updates the zetaclient.
I don't think the regular start-upgrade-test-zetaclient test is needed at all

Reply:
Acually, the zetaclient upgrade tests will not work at all. 
v38 zetaclient is not compatible with v36 zetacore , whcih is what the test basically checks 

This is mainly becuase of the changes in proto files for ChainParams 
https://github.com/zeta-chain/zeta-node/blob/382393b1370907681e3dd1c0e858773c9a7ae8df/zetaclient/orchestrator/contextupdater.go#L101-L107

The new zetaclient uses the new validate which fails if validating older chain params 

I think this is fine , we dont need to enable zetaclient upgrade tests for this release